src/modules/fancyzones/FancyZonesLib/FancyZones.cpp

```diff
@@ -1241,12 +1240,6 @@ void FancyZones::OnSettingsChanged() noexcept
     // Needed if we toggled spanZonesAcrossMonitors
     m_workAreaHandler.Clear();
 
-    // update zone colors
-    m_workAreaHandler.UpdateZoneColors(GetZoneColors());
```

Comment:
this should be fixed, not deleted.

Reply:
What do you mean by fix?

As I see it, the state was cleared, so it is not necessary to update.

Reply:
I reverted this change.